app/src/main/java/org/simple/clinic/storage/StorageModule.kt

```diff
@@ -35,9 +35,9 @@ class StorageModule {
     // internally.
     val sqliteThreadPoolCount = SQLiteGlobal.getWALConnectionPoolSize() / 2
     val queryExecutor = ThreadPools.create(
-        corePoolSize = 1,
+        corePoolSize = sqliteThreadPoolCount,
         maxPoolSize = sqliteThreadPoolCount,
-        threadPrefix = "room-query-"
+        threadPrefix = "room-query"
```

Comment:
Any particular reason for removing the hyphen from the thread prefix? 🤔

Reply:
The reason is explained in the commit message.

> Remove '-' from the Room thread name prefix
>
> This was done because the `ThreadPools` factory method already adds its own '-' where needed.****

Reply:
Oh, I missed it. Sorry 😅